appveyor.yml

```diff
@@ -79,4 +79,4 @@ install:
 build: off
 
 test_script:
-    - '%CMD_IN_ENV% obvci_conda_build_dir .\ odm2 --channel main  --build-condition "numpy >=1.9" "%PY_CONDITION%"'
+    - '%CMD_IN_ENV% conda-build-all .\ --upload-channels odm2 --matrix-conditions "numpy >=1.9" "%PY_CONDITION%"'
```

Comment:
Testing `conda-build-all` (new `obvious-ci`) so we can use latest `conda-build`.